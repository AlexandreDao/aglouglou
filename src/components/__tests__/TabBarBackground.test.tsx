import React from 'react'
import { render } from '@testing-library/react-native'
import BlurTabBarBackgroundAndroid from '@/components/TabBarBackground.android'
import BlurTabBarBackgroundIOS from '@/components/TabBarBackground.ios'

describe('<TabBarBackground />', () => {
  test('TabBarBackground snapshot (Android)', () => {
    expect(BlurTabBarBackgroundAndroid).toBeUndefined()
  })

  test('TabBarBackground snapshot (iOS)', () => {
    const tree = render(<BlurTabBarBackgroundIOS />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
