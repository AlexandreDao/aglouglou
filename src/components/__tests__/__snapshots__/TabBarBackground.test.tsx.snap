// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TabBarBackground /> TabBarBackground snapshot (iOS) 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "transparent",
      },
      [
        {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        {
          "backgroundColor": "rgba(67, 19, 39, 0.6)",
        },
      ],
    ]
  }
>
  <ViewManagerAdapter_ExpoBlurView
    blurReductionFactor={4}
    experimentalBlurMethod="none"
    intensity={80}
    style={
      {
        "bottom": 0,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
    tint="default"
  />
</View>
`;
